_posts/2020-04-15-Cleard.md

```diff
@@ -0,0 +1,65 @@
+---
+author: John
+title: Week four co-winner Get Cleard, works to improve global screening processes
+---
+
+![Get Cleard](images/blog/Get Cleard.png)
+
+This week’s competition resulted in a tie-a first for our Codevid-19 global hackathon. Jaime Wood kindly offered to answer a few questions about her team’s entry, [Get Cleard](https://www.cleard.ai/).
```

Comment:
The dash here could either be an em-dash followed by a space: —

...or just your plain old colon and a space.  Right now it just looks like we've invented a new word called "tie-a" :)

Reply:
Yes indeed it does look like some serious coinage as the grammarians say.  These weird little syntax things are stupid hard for me to catch right now.  That is about the last place I'd look for a prob., but of course...syntax.